nixos/modules/services/misc/mesos-slave.nix

```diff
@@ -22,8 +24,14 @@ in {
         type = types.uniq types.bool;
       };
 
+      ip = mkOption {
+        description = "IP address to listen on.";
+        default = "0.0.0.0";
+        type = types.string;
+      };
+
       port = mkOption {
-        description = "Mesos Slave port";
+        description = "Port to listen on.";
```

Comment:
Thought mentioning "Mesos Slave" in descriptions in the mesos-slave service was redundant but not something I feel strongly about. Can revert.